src/data-workspace/inputs/validators.test.js

```diff
@@ -0,0 +1,66 @@
+import { VALUE_TYPES, CAN_HAVE_LIMITS_TYPES } from '../../shared/index.js'
+import { validateByValueTypeWithLimits } from './validators.js'
+
+describe(`validateByValueTypeWithLimits`, () => {
+    describe(`when limits are defined`, () => {
```

Comment:
I think my main concern here was that the test also is covering the implementation of `createNumberRange` which comes from the UI library - I could mock this but then I general prefer to test the code in a way that would reflect real output for a user and also I think there is generally better value in testing the real code rather than overly relying on mocks